src/test/java/org/fundacionjala/sfdc/hooks/Hooks.java

```diff
@@ -2,6 +2,9 @@
 
 
 import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import io.qameta.allure.Allure;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.qameta.allure.Allure'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)